### ISSUE TITLE
[Feature Request] External Overlay

### ISSUE THREAD

**Issue by suffz** [AllenDang/giu#930]
### Your request

I was just wondering if its currently possible to overlay a imgui over a different process using this library

**Comment by gucio321** [comment]
What do you mean by overlay on different prices? Write guy UI in another application or write a standalone transparent app

**Comment by suffz** [comment]
have the imgui render onto another application just like a overlay.

**Comment by gucio321** [comment]
Yes, this is possibly for cimgui go with ebiten apps and giu is on the right way to have this as well. Just a few changes with Context.

**Comment by suffz** [comment]
do you have any current examples for this in cimgui?

**Comment by gucio321** [comment]
Yes, here: https://github.com/AllenDang/cimgui-go/tree/main/examples%2Febiten-game

**Comment by suffz** [comment]
So I'll be able to overlay onto Notepad.exe following this code? We'll get a idea

**Comment by gucio321** [comment]
well, if you want to add cimgui overlay to the app you don't have source code access OR which is not writen in ebiten you cant use code I linked above.
In this case my suggestion would be checking out transparent example (but this may require glfw 3.4 for mouse passthrough #559) OR exploring imgui viewports https://github.com/ocornut/imgui/wiki/Multi-Viewports

**Comment by Pugnator** [comment]
> [@suffz](https://github.com/suffz) check out examples/transparent - it does whot you need imo.
> 
> Ping me here if you need further assistance.

I don't know how to achieve the same effect with multiple windows. My app is triggered via hotkeys, and one of the actions opens a semi-transparent window where the user selects an area with the mouse. While it works in SingleWindow mode, I can't get it to work with a multiple-window setup. It seems like the alpha channel is not active at all.
